fe/src/main/java/org/apache/doris/analysis/FunctionCallExpr.java

```diff
@@ -605,6 +605,19 @@ public void analyzeImpl(Analyzer analyzer) throws AnalysisException {
             return;
         }
 
+        if (fnName.getFunction().equalsIgnoreCase("append_trailing_char_if_absent")) {
```

Comment:
I think this check is unnecessary, can be removed.
If you keep it, this will make this function more complicate.

Reply:
Yes，I understand there will make this function more complicate, but  I need to avoid two parameters be Non-String type，such as `append_trailing_char_if_absent(1,2)`, and I don't known how to handle this case if i don't check here. Maybe there is a better way to handle this case if you known.

Reply:
For this case you mentioned, Doris can cast input argument to function's definition type. So you don't handle this in this place.

Reply:
ok，Done